manifests/plugin/proxmox.pp

```diff
@@ -0,0 +1,5 @@
+# = Foreman Proxmox plugin
+class foreman::plugin::proxmox {
+  foreman::plugin { 'proxmox':
```

Comment:
I think the actual package is called `fog_proxmox` so it would probably fail to really install. Right now we don't have a test with actual package (RPM/deb) installations, so it's hard to verify with automated tests now.